packages/react-components/src/templates/__tests__/Layout.test.tsx

```diff
@@ -75,6 +75,8 @@ it('closes the drawer when clicking the overlay', async () => {
 });
 
 it('closes the drawer on navigation', async () => {
+  Element.prototype.scrollTo = jest.fn();
```

Comment:
Maybe this should be moved to the jest setup file?

Reply:
Don't think so, we usually only mock what's necessary for a specific test